atef/widgets/config/run_base.py

```diff
@@ -251,32 +252,101 @@ def setup_buttons(self, configs, next_widget: TreeItem = None) -> None:
             self.setup_next_button(next_widget)
 
         self.setup_verify_button()
+        self.abort_button.clicked.connect(self.abort_task)
+        self.reveal_run_or_abort(running=False)
+
+    def reveal_run_or_abort(self, running: bool):
+        if running:
+            self.abort_button.show()
+            self.run_button.hide()
+        else:
+            self.abort_button.hide()
+            self.run_button.show()
```

Comment:
General UX thoughts: buttons that do different things but nominally occupy the same space on the screen via show/hide/replace can be disruptive to muscle memory.